frappe/public/js/frappe/form/form.js

```diff
@@ -837,6 +839,8 @@ frappe.ui.form.Form = class FrappeForm {
 		frappe.call('frappe.desk.form.utils.get_next', args).then(r => {
 			if (r.message) {
 				frappe.set_route('Form', this.doctype, r.message);
+				let $first_input_el = $(frappe.container.page).find('.frappe-control:visible').eq(0);
```

Comment:
Better to use abstract this kind of functionality into a `focus_on_first_input` method and then use it here.